packages/main/src/TabContainer.hbs

```diff
@@ -11,6 +11,7 @@
 			<div class="{{classes.headerBackArrow}}">
 				<ui5-button @click="{{_onHeaderBackArrowClick}}"
 							icon="slim-arrow-left"
+							data-ui5-stable="tabsLeft"
```

Comment:
**[suggestion (non-blocking)](https://conventionalcomments.org/#labels):** Should we be using kebab case for the `"tabsLeft"`, `"tabsRight"`, and `"tabsOverflow"` values of the newly introduced `data-ui5-stable` attributes in this file?